website/docs/docs/dbt-versions/release-notes/09-April-2023/scheduler-optimized.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "dbt Cloud more intelligently processes scheduled jobs"
+id: "scheduler-optimized"
+description: "April 2023: "
+sidebar_label: "Update: Scheduler optimizes job queue"
+tags: [Apr-12-2023, scheduler]
+---
+
+The dbt Cloud Scheduler now prevents queue clog by canceling queued runs in excess of 50. The job queue can build up when the duration of a job run grows over time, usually caused by more data landing in your cloud warehouse. Once the run duration becomes longer than the frequency of the job’s schedule, runs start to queue, waiting for the long running job to finish. 
+
+Previously, when you scheduled a job to run more frequently than it takes to complete a single run, the job queue would get backed up, and the scheduler couldn't process all the runs, so it would start canceling runs indiscriminately. You used to have to go into dbt Cloud to manually cancel all the queued runs and change the job schedule to "unclog" the scheduler queue.
+
+The dbt Cloud scheduler now detects when a job is in this clogged state, and manages the queue to run more optimally -- providing a helpful message when the schedluer has taken action.
```

Comment:
fix typo

```suggestion
The dbt Cloud scheduler now detects when a job is in this clogged state, and manages the queue to run more optimally -- providing a helpful message when the scheduler has taken action.
```

Reply:
Based on the release note, i think we should include something about how only one job can be queued at a time - so for example, the below can be added after the word 'optimally.':

Only one run of the job can be queued at a time, and if a newer run is queued, the older one will be canceled with an error message.